decidim-elections/config/i18n-tasks.yml

```diff
@@ -5,6 +5,7 @@ locales: [en]
 
 ignore_unused:
   - "decidim.components.elections.name"
+  - decidim.elections.admin_log.*
```

Comment:
Why are you adding all decidim.elections.admin_log translations here ? Is there some unused translation?

Reply:
Hi! In `election_presenter.rb` there's a tag that needs to be excluded from the checks `"decidim.elections.admin_log.election.#{action}"`

Reply:
This change now is done in `config/i18n-tasks.yml` because in `decidim-elections/config/i18n-tasks.yml` was not having effect.

Reply:
Ok, as @agustibr said, this must be addressed in static way please:
![image](https://user-images.githubusercontent.com/9702463/87757876-b2e88600-c80b-11ea-9464-a41e42b573fc.png)
instead of:
![image](https://user-images.githubusercontent.com/9702463/87757915-c5fb5600-c80b-11ea-9312-d7e92306f985.png)
Thanks!